content/guides/embed/ui-elements/installation.md

```diff
@@ -12,20 +12,8 @@ alias_paths: []
 # Installation
 
 UI Elements can be used either by downloading the Javascript libraries
-directly or by pulling from our NPM packages. All UI Elements also
-require their corresponding CSS stylesheet to render properly.
-
-All of the UI elements are available both through [NPM][npm] as well as for
-direct download.
-
-<Message>
-  There are two ways to use the Box UI elements. If you’re looking to build
-  something quickly, use it as a library as shown below in this
-  documentation. Alternatively, if you are a building a React based app, you can
-  pull in the component from our NPM package. For details refer to the NPM link
-  above. As we continue to roll this out, we will provide some level of access
-  to the source.
-</Message>
```

Comment:
I was originally going to remove the last two lines of this paragraph:

- the last line is outdated, all the code is open source
- the second to last line seems unnecessary because of the section directly above and below it

but then I ended up removing the entire paragraph because it seems to just repeat the same information as above and below it